app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -217,10 +217,10 @@ async def raster_tile_cache_validator(
     symbology_type = input_data["creation_options"].get("symbology", {}).get("type")
     if symbology_type:
         symbology_info = symbology_constructor[symbology_type]
-        req_input_bands: Optional[int] = symbology_info.req_input_bands
+        req_input_bands: Optional[List[int]] = symbology_info.req_input_bands
 
         if req_input_bands and (
-            req_input_bands != source_asset.creation_options.get("band_count", 1)
+            req_input_bands in source_asset.creation_options.get("band_count", [1])
```

Comment:
When the `int` `band_count` is passed, don't we have to transform it to the list of band ids here?

Reply:
Also, will the `SymbologyInfo` `req_input_bands` arguments have to be updated to lists here: https://github.com/wri/gfw-data-api/blob/master/app/tasks/raster_tile_cache_assets/symbology.py#L645?

Reply:
@solomon-negusse whoops, I got the check backwards! Fixed